plugins/Morpheus/javascripts/ajaxHelper.js

```diff
@@ -529,10 +529,16 @@ function ajaxHelper() {
      */
     this._mixinDefaultGetParams = function (params) {
 
+        if (window.location.hash) {
```

Comment:
If the hash isn't a valid query string (or has no segment), should we default to the segment in the URL?

Reply:
thought about it. but wasn't sure about any side effects. e.g. opening the dashboard with having a segment in the query string and in the hash. If it would be removed from the hash it would still load the one in query string which might not be correct/wanted

Reply:
Hmm, not sure about that. Thought this was how it worked for other query params (like date/period), but maybe not... cc @mattab

Reply:
Don't think the segment of the query string should be used if the one from the hash gets removed (be choosing no segment in UI)